### PR TITLE
Reminder emails for unregistered mentors/mentees

### PR DESCRIPTION
**Pull request by anjali-sharma**

### CODE REVIEW

File: lib/tasks/registration_reminder.rake [hunk 1]
```diff
@@ -0,0 +1,19 @@
+namespace :registration_reminder do
+  
+  desc "send email reminder to unregistered mentors"
+  task mentor_registration_reminder: :environment do
+  	unregistered_mentors = User.where("role=? AND registration_token IS NOT NULL", User.roles[:mentor])
```
Comment (DMips) [inline lib/tasks/registration_reminder.rake]: you can use here User.mentor -> that will return all users with role = mentor

File: lib/tasks/registration_reminder.rake [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  	unregistered_mentors.each do |mentor|
+  		UserRegistrations.user_registration_reminder(mentor).deliver
+  	end
+  end
+
+  desc "send email reminder to unregistered mentees"
+  task mentee_registration_reminder: :environment do
+  	unregistered_mentees = User.where("role=? AND registration_token IS NOT NULL", User.roles[:mentee])
```
Comment (DMips) [inline lib/tasks/registration_reminder.rake]: you can use here User.mentee -> that will return all users with role = mentee

File: app/views/user_registrations/user_registration_reminder.html.slim [hunk 3]
```diff
@@ -0,0 +1,22 @@
+p
+  => "Dear"
```
Comment (DMips) [inline app/views/user_registrations/user_registration_reminder.html.slim]: can we use email templates here? so organizers will be able to edit message from the website?
you need to add here https://github.com/LearnITGirl/WebPlatform/blob/master/app/models/email_template.rb#L2 new keys like `unregistered_mentee: 13` and `unregistered_mentor: 14`. Add new cases here https://github.com/LearnITGirl/WebPlatform/blob/master/app/models/email_template.rb#L26 with correct scopes, create templates (you can use /email_templates/new) and put them in the seed. if you won't have time for this let me know and i'll do it :)

Comment (anjali-sharma) [inline app/views/user_registrations/user_registration_reminder.html.slim]: sounds good. I'll do it now.

**Comment by DMips** [comment]
@anjali-sharma looks good :)

**Pull request merged**